paper/paper.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'scikit-optimize: Sequential model-based optimization toolkit.'
+tags:
+  - Python
+  - Machine-learning
+  - Global optimization
+  - Scientific Computing
+authors:
+ - name: Manoj Kumar
+   orcid: 0000-0002-3802-8346
+   affiliation: 1
+ - name: Tim Head
+   orcid:
+   affiliation: 2
+ - name: Gilles Louppe
+   orcid:
+   affiliation: 3
+ - name: Katie Malone
+   orcid:
+   affiliation: 4
+ - name: nel215
+   orcid:
+   affliation: 5
+ - name: Nuno Campos
+   orcid:
+   affliation: 6
+ - name: Zé Vinícius
+   orcid:
+   affliation: 7
+ - name: Thomas Fan
+   orcid:
+   affliation: 8
+ - name: Mikhail Pak
+   orcid:
+   affliation: 9
+ - name: Iaroslav Shcherbatyi
+   orcid:
+   affliation: 10
+ - name: Lilian Besson
+   orcid:
+   affliation: 11
+ - name: Fabian Linzberger
+   orcid:
+   affliation: 12
+ - name: Taylor Smith
+   orcid:
+   affliation: 13
+ - name: andreh7
+   orcid:
+   affliation: 14
+ - name: Mehdi Cherti
+   orcid:
+   affliation: 15
+ - name: Alexander Fabisch
+   orcid:
+   affliation: 16
+
+affiliations:
+ - name: New York University
+   index: 1
+ - name:
+   index: 2
+ - name:
+   index: 3
+ - name:
+   index: 4
+ - name:
+   index: 5
+ - name:
+   index: 6
+ - name:
+   index: 7
+ - name:
+   index: 8
+ - name:
+   index: 9
+ - name:
+   index: 10
+ - name:
+   index: 11
+ - name:
+   index: 12
+ - name:
+   index: 13
+ - name:
+   index: 14
+ - name:
+   index: 15
+ - name:
+   index: 16
+date: June 2, 2017
+bibliography: paper.bib
+---
+
+# Summary
+scikit-optimize is a sequential model-based optimization toolkit.
+
+scikit-optimize aims at the global minimization of functions that are noisy,
+expensive or for which gradients are not available. Examples of such optimization
+includes optimizing the loss of a deep neural network on validation-data or
+the settings of a robot controller. The technique used by
+scikit-optimize is popularly known as sequential optimization or
+bayesian optimization. The key idea behind this technique is
+that a predictive model is used to suggest a new candidate point given the
```

Comment:
Here one might understand this as if predictive model is used whose output is the next point which will be evaluated. 

Consider using something along the lines: "At every iteration, predictive model is trained on the set of points which has been already evaluated to estimate the value of objective given some new point. A special class of models is used, which can give an 'exploration' score, which indicates how close the point is to already evaluated points (in some sense). Then the next point for evaluation is such that it minimizes a tradeoff between the estimated objective value and 'exploration' score. "